src/content/getting-started/designers/designers.md

```diff
@@ -2,16 +2,38 @@
 title: Designers
 ---
 
-This site, along with the Carbon Design Kit, provides designers with everything they need to get up and running with Carbon.
+## Resources
+
+<grid-wrapper col_lg="8" flex="true" bleed="true">
+<clickable-tile
+    title="Carbon Design Kit"
+    href="https://github.com/ibm/carbon-design-kit"
+    type="resource"
+     >
+    <img src="images/sketch-icon.png" alt="Carbon Design Kit"  />
+</clickable-tile>
+<clickable-tile
+    title="Download Box Drive."
```

Comment:
Change to "Connect with Box" add a second line saying "IBMers only"

Reply:
f we don't change the title of the section then we need to say "Connect the kit through Box"